libqtile/widget/base.py

```diff
@@ -648,12 +654,22 @@ def draw(self):
 
         if not self.bar.horizontal and self.rotate:
             # Left bar reads bottom to top
-            if self.bar.screen.left is self.bar:
+            # Can be overriden to read bottom to top all the time with vertical_text_direction
+            if (
+                self.bar.screen.left is self.bar
+                and self.vertical_text_direction == "default"
+                or self.vertical_text_direction == "bottom_to_top"
```

Comment:
```suggestion
                (self.bar.screen.left is self.bar
                and self.vertical_text_direction == "default")
                or self.vertical_text_direction == "bottom_to_top"
```
I think some parentheses might help readability here.